platformio.ini

```diff
@@ -190,8 +190,6 @@ src_build_flags =
 build_flags =
   ${common.build_flags}
   ${common.build_flags_esp32}
-  -DRX1=25
-  -DTX1=27
```

Comment:
Think these are removed by mistake

Reply:
Oops. I removed these because none of the code seems to use them :-)

Reply:
They are used in the ESP32 BSP to set the Serial pins, so you won't find the references in the code, just overrides the defaults.